.travis.yml

```diff
@@ -13,7 +13,9 @@ env:
 install:
 - "pip install -r requirements-test.txt"
 - $INSTALLATION
-- pip install pytest_dbfixtures[mongodb,redis,rabbitmq,mysql,postgresql,elasticsearch,tests] coveralls wheel
+- pip install pytest_dbfixtures[mongodb,redis,rabbitmq,mysql,postgresql,elasticsearch,dynamodb,tests] coveralls wheel
+- mkdir /tmp/dynamodb
+- wget -O - http://dynamodb-local.s3-website-us-west-2.amazonaws.com/dynamodb_local_latest | tar xz --directory /tmp/dynamodb
```

Comment:
>  is there no deb installation?

I have not found it. I have found this for osx (install by `brew`), but...

> Put it as a first step, and shell script for pre-installation would be good as well. then these two lines could be documented. why wget?
What kind of information should I put there?